svelte-development/src/MapPoints.svelte

```diff
@@ -123,29 +141,52 @@
       {@html butterflies[1]}
     </g>
   </defs>
-  {#each $butterflyPoints as {x, y, value, regionIndex, regionShape}}
+
+  <g class="link-lines">
```

Comment:
@Yuriko-Schumacher  we should be using a line generator instead of hardcoding svg code.